.github/ISSUE_TEMPLATE/ui_bug_report.md

```diff
@@ -1,8 +1,8 @@
 ---
-name: Bug report
-about: File a bug report
+name: 'UI: Bug report'
```

Comment:
do these templates work if the .github/ folder is not in the root directory?

Reply:
Unfortunately not AFAIK, they weren't working after merge, so moved the UI Bug report template down to the root dir and got rid of the UI feature request for now until we come up with a better structure around how feature requests should be presented